src/style.css

```diff
@@ -1,3 +1,55 @@
 body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.img {
+  margin-left: 50px;
+}
+
+.nav {
+
+  display: flex;
+  height: 60px;
```

Comment:
the height should be set for nav links (not the header or navigation), take it from the design.